esrally/track/params.py

```diff
@@ -621,12 +621,16 @@ def __init__(self, track, params, **kwargs):
         self._index_name = params.get("index", default_index)
         self._max_num_segments = params.get("max-num-segments")
         self._request_timeout = params.get("request-timeout")
+        self._poll_period = params.get("poll-period", 1)
```

Comment:
Isn't the default `poll-period` supposed to be `10s`?

Reply:
Well spotted @dliappis. Bumped to 10. This is maybe a bit conservative but the users can always configure.

Reply:
Maybe we could have a tiny unittest to ensure this gets checked properly?